azkaban-common/src/test/java/azkaban/test/Utils.java

```diff
@@ -37,13 +37,8 @@ public static DatabaseOperator initTestDB() throws Exception{
     final Props props = new Props();
     props.put("database.sql.scripts.dir", sqlScriptsDir);
 
-    // TODO kunkun-tang: Need to refactor AzkabanDatabaseSetup to accept datasource in azkaban-db
-    final azkaban.database.AzkabanDataSource dataSourceForSetupDB =
-        new azkaban.database.AzkabanConnectionPoolTest.EmbeddedH2BasicDataSource();
-    final AzkabanDatabaseSetup setup = new AzkabanDatabaseSetup(dataSourceForSetupDB, props);
-    setup.loadTableInfo();
-    setup.updateDatabase(true, false);
-
+    final DatabaseSetup setup = new DatabaseSetup(dataSource, sqlScriptsDir);
+    setup.updateDatabase();
```

Comment:
In the new code, are we not doing loadTableInfo()? And inside updateDatabase(), it seems there is no updateTables(). Are they removed intentionally?